app/models/comparison/metric_type.rb

```diff
@@ -0,0 +1,29 @@
+# == Schema Information
+#
+# Table name: comparison_metric_types
+#
+#  created_at :datetime         not null
+#  fuel_type  :integer          not null
+#  id         :bigint(8)        not null, primary key
+#  key        :string           not null
+#  units      :integer          not null
+#  updated_at :datetime         not null
+#
+class Comparison::MetricType < ApplicationRecord
+  self.table_name = 'comparison_metric_types'
+
+  include EnumFuelTypeable
+  extend Mobility
+
+  translates :label, type: :string, fallbacks: { cy: :en }
+  translates :description, type: :string, fallbacks: { cy: :en }
+
+  validates :key, presence: true, uniqueness: true
```

Comment:
Needs to be scoped to the fuel type? E.g. we'll have say `annual_kwh` for gas, electricity, etc.